Traceback (most recent call last):
  File "/home/minseokhwan/xray_projects/xray_scattering_discrimination/run_data_collection.py", line 339, in <module>
    os.mkdir(directory + "/data/" + identifier)
FileNotFoundError: [Errno 2] No such file or directory: '/home/minseokhwan/xray_projects/xray_scattering_discrimination/data/YAGCe_150keV'
TypeError: only length-1 arrays can be converted to Python scalars

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/minseokhwan/xray_projects/xray_scattering_discrimination/run_data_collection.py", line 362, in <module>
    simulation.collect_simulation_data(Nreps=100, Nruns_per_energy=1000)
  File "/home/minseokhwan/xray_projects/xray_scattering_discrimination/run_data_collection.py", line 324, in collect_simulation_data
    self.read_output(Nruns_per_energy)
  File "/home/minseokhwan/xray_projects/xray_scattering_discrimination/run_data_collection.py", line 254, in read_output
    dE_trans[i] = E_inc - E_trans
ValueError: setting an array element with a sequence.
TypeError: only length-1 arrays can be converted to Python scalars

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/minseokhwan/xray_projects/xray_scattering_discrimination/run_data_collection.py", line 362, in <module>
TypeError: only length-1 arrays can be converted to Python scalars

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/minseokhwan/xray_projects/xray_scattering_discrimination/run_data_collection.py", line 362, in <module>
TypeError: only length-1 arrays can be converted to Python scalars

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/minseokhwan/xray_projects/xray_scattering_discrimination/run_data_collection.py", line 362, in <module>
TypeError: only length-1 arrays can be converted to Python scalars

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/minseokhwan/xray_projects/xray_scattering_discrimination/run_data_collection.py", line 362, in <module>
TypeError: only length-1 arrays can be converted to Python scalars

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/minseokhwan/xray_projects/xray_scattering_discrimination/run_data_collection.py", line 362, in <module>
    simulation.collect_simulation_data(Nreps=100, Nruns_per_energy=1000)
  File "/home/minseokhwan/xray_projects/xray_scattering_discrimination/run_data_collection.py", line 324, in collect_simulation_data
    simulation.collect_simulation_data(Nreps=100, Nruns_per_energy=1000)
  File "/home/minseokhwan/xray_projects/xray_scattering_discrimination/run_data_collection.py", line 324, in collect_simulation_data
    simulation.collect_simulation_data(Nreps=100, Nruns_per_energy=1000)
  File "/home/minseokhwan/xray_projects/xray_scattering_discrimination/run_data_collection.py", line 324, in collect_simulation_data
    self.read_output(Nruns_per_energy)
  File "/home/minseokhwan/xray_projects/xray_scattering_discrimination/run_data_collection.py", line 254, in read_output
    simulation.collect_simulation_data(Nreps=100, Nruns_per_energy=1000)
  File "/home/minseokhwan/xray_projects/xray_scattering_discrimination/run_data_collection.py", line 324, in collect_simulation_data
TypeError: only length-1 arrays can be converted to Python scalars

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/minseokhwan/xray_projects/xray_scattering_discrimination/run_data_collection.py", line 362, in <module>
TypeError: only length-1 arrays can be converted to Python scalars

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/minseokhwan/xray_projects/xray_scattering_discrimination/run_data_collection.py", line 362, in <module>
TypeError: only length-1 arrays can be converted to Python scalars

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/minseokhwan/xray_projects/xray_scattering_discrimination/run_data_collection.py", line 362, in <module>
    simulation.collect_simulation_data(Nreps=100, Nruns_per_energy=1000)
  File "/home/minseokhwan/xray_projects/xray_scattering_discrimination/run_data_collection.py", line 324, in collect_simulation_data
TypeError: only length-1 arrays can be converted to Python scalars

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/minseokhwan/xray_projects/xray_scattering_discrimination/run_data_collection.py", line 362, in <module>
    self.read_output(Nruns_per_energy)
  File "/home/minseokhwan/xray_projects/xray_scattering_discrimination/run_data_collection.py", line 254, in read_output
TypeError: only length-1 arrays can be converted to Python scalars

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/minseokhwan/xray_projects/xray_scattering_discrimination/run_data_collection.py", line 362, in <module>
    self.read_output(Nruns_per_energy)
  File "/home/minseokhwan/xray_projects/xray_scattering_discrimination/run_data_collection.py", line 254, in read_output
    simulation.collect_simulation_data(Nreps=100, Nruns_per_energy=1000)
  File "/home/minseokhwan/xray_projects/xray_scattering_discrimination/run_data_collection.py", line 324, in collect_simulation_data
    dE_trans[i] = E_inc - E_trans
ValueError: setting an array element with a sequence.
    simulation.collect_simulation_data(Nreps=100, Nruns_per_energy=1000)
  File "/home/minseokhwan/xray_projects/xray_scattering_discrimination/run_data_collection.py", line 324, in collect_simulation_data
    self.read_output(Nruns_per_energy)
  File "/home/minseokhwan/xray_projects/xray_scattering_discrimination/run_data_collection.py", line 254, in read_output
    simulation.collect_simulation_data(Nreps=100, Nruns_per_energy=1000)
  File "/home/minseokhwan/xray_projects/xray_scattering_discrimination/run_data_collection.py", line 324, in collect_simulation_data
TypeError: only length-1 arrays can be converted to Python scalars

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/minseokhwan/xray_projects/xray_scattering_discrimination/run_data_collection.py", line 362, in <module>
    self.read_output(Nruns_per_energy)
  File "/home/minseokhwan/xray_projects/xray_scattering_discrimination/run_data_collection.py", line 254, in read_output
    simulation.collect_simulation_data(Nreps=100, Nruns_per_energy=1000)
  File "/home/minseokhwan/xray_projects/xray_scattering_discrimination/run_data_collection.py", line 324, in collect_simulation_data
    dE_trans[i] = E_inc - E_trans
ValueError: setting an array element with a sequence.
    simulation.collect_simulation_data(Nreps=100, Nruns_per_energy=1000)
  File "/home/minseokhwan/xray_projects/xray_scattering_discrimination/run_data_collection.py", line 324, in collect_simulation_data
    self.read_output(Nruns_per_energy)
  File "/home/minseokhwan/xray_projects/xray_scattering_discrimination/run_data_collection.py", line 254, in read_output
    dE_trans[i] = E_inc - E_trans
ValueError: setting an array element with a sequence.
    self.read_output(Nruns_per_energy)
  File "/home/minseokhwan/xray_projects/xray_scattering_discrimination/run_data_collection.py", line 254, in read_output
    dE_trans[i] = E_inc - E_trans
ValueError: setting an array element with a sequence.
    dE_trans[i] = E_inc - E_trans
ValueError: setting an array element with a sequence.
    simulation.collect_simulation_data(Nreps=100, Nruns_per_energy=1000)
  File "/home/minseokhwan/xray_projects/xray_scattering_discrimination/run_data_collection.py", line 324, in collect_simulation_data
    self.read_output(Nruns_per_energy)
  File "/home/minseokhwan/xray_projects/xray_scattering_discrimination/run_data_collection.py", line 254, in read_output
    dE_trans[i] = E_inc - E_trans
ValueError: setting an array element with a sequence.
    self.read_output(Nruns_per_energy)
  File "/home/minseokhwan/xray_projects/xray_scattering_discrimination/run_data_collection.py", line 254, in read_output
    dE_trans[i] = E_inc - E_trans
ValueError: setting an array element with a sequence.
    self.read_output(Nruns_per_energy)
  File "/home/minseokhwan/xray_projects/xray_scattering_discrimination/run_data_collection.py", line 254, in read_output
TypeError: only length-1 arrays can be converted to Python scalars

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/minseokhwan/xray_projects/xray_scattering_discrimination/run_data_collection.py", line 362, in <module>
    dE_trans[i] = E_inc - E_trans
ValueError: setting an array element with a sequence.
TypeError: only length-1 arrays can be converted to Python scalars

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/minseokhwan/xray_projects/xray_scattering_discrimination/run_data_collection.py", line 362, in <module>
    self.read_output(Nruns_per_energy)
  File "/home/minseokhwan/xray_projects/xray_scattering_discrimination/run_data_collection.py", line 254, in read_output
    dE_trans[i] = E_inc - E_trans
ValueError: setting an array element with a sequence.
    dE_trans[i] = E_inc - E_trans
ValueError: setting an array element with a sequence.
    simulation.collect_simulation_data(Nreps=100, Nruns_per_energy=1000)
  File "/home/minseokhwan/xray_projects/xray_scattering_discrimination/run_data_collection.py", line 324, in collect_simulation_data
    simulation.collect_simulation_data(Nreps=100, Nruns_per_energy=1000)
  File "/home/minseokhwan/xray_projects/xray_scattering_discrimination/run_data_collection.py", line 324, in collect_simulation_data
    dE_trans[i] = E_inc - E_trans
ValueError: setting an array element with a sequence.
    self.read_output(Nruns_per_energy)
  File "/home/minseokhwan/xray_projects/xray_scattering_discrimination/run_data_collection.py", line 254, in read_output
    self.read_output(Nruns_per_energy)
  File "/home/minseokhwan/xray_projects/xray_scattering_discrimination/run_data_collection.py", line 254, in read_output
    dE_trans[i] = E_inc - E_trans
ValueError: setting an array element with a sequence.
    dE_trans[i] = E_inc - E_trans
ValueError: setting an array element with a sequence.
TypeError: only length-1 arrays can be converted to Python scalars

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/minseokhwan/xray_projects/xray_scattering_discrimination/run_data_collection.py", line 362, in <module>
    simulation.collect_simulation_data(Nreps=100, Nruns_per_energy=1000)
  File "/home/minseokhwan/xray_projects/xray_scattering_discrimination/run_data_collection.py", line 324, in collect_simulation_data
    self.read_output(Nruns_per_energy)
  File "/home/minseokhwan/xray_projects/xray_scattering_discrimination/run_data_collection.py", line 254, in read_output
    dE_trans[i] = E_inc - E_trans
ValueError: setting an array element with a sequence.
TypeError: only length-1 arrays can be converted to Python scalars

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/minseokhwan/xray_projects/xray_scattering_discrimination/run_data_collection.py", line 362, in <module>
TypeError: only length-1 arrays can be converted to Python scalars

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/minseokhwan/xray_projects/xray_scattering_discrimination/run_data_collection.py", line 362, in <module>
    simulation.collect_simulation_data(Nreps=100, Nruns_per_energy=1000)
  File "/home/minseokhwan/xray_projects/xray_scattering_discrimination/run_data_collection.py", line 324, in collect_simulation_data
    self.read_output(Nruns_per_energy)
  File "/home/minseokhwan/xray_projects/xray_scattering_discrimination/run_data_collection.py", line 254, in read_output
    simulation.collect_simulation_data(Nreps=100, Nruns_per_energy=1000)
  File "/home/minseokhwan/xray_projects/xray_scattering_discrimination/run_data_collection.py", line 324, in collect_simulation_data
    dE_trans[i] = E_inc - E_trans
ValueError: setting an array element with a sequence.
    self.read_output(Nruns_per_energy)
  File "/home/minseokhwan/xray_projects/xray_scattering_discrimination/run_data_collection.py", line 254, in read_output
TypeError: only length-1 arrays can be converted to Python scalars

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/minseokhwan/xray_projects/xray_scattering_discrimination/run_data_collection.py", line 362, in <module>
TypeError: only length-1 arrays can be converted to Python scalars

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/minseokhwan/xray_projects/xray_scattering_discrimination/run_data_collection.py", line 362, in <module>
    dE_trans[i] = E_inc - E_trans
ValueError: setting an array element with a sequence.
    simulation.collect_simulation_data(Nreps=100, Nruns_per_energy=1000)
  File "/home/minseokhwan/xray_projects/xray_scattering_discrimination/run_data_collection.py", line 324, in collect_simulation_data
    simulation.collect_simulation_data(Nreps=100, Nruns_per_energy=1000)
  File "/home/minseokhwan/xray_projects/xray_scattering_discrimination/run_data_collection.py", line 324, in collect_simulation_data
    self.read_output(Nruns_per_energy)
  File "/home/minseokhwan/xray_projects/xray_scattering_discrimination/run_data_collection.py", line 254, in read_output
    self.read_output(Nruns_per_energy)
  File "/home/minseokhwan/xray_projects/xray_scattering_discrimination/run_data_collection.py", line 254, in read_output
    dE_trans[i] = E_inc - E_trans
ValueError: setting an array element with a sequence.
    dE_trans[i] = E_inc - E_trans
ValueError: setting an array element with a sequence.
TypeError: only length-1 arrays can be converted to Python scalars

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/minseokhwan/xray_projects/xray_scattering_discrimination/run_data_collection.py", line 362, in <module>
TypeError: only length-1 arrays can be converted to Python scalars

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/minseokhwan/xray_projects/xray_scattering_discrimination/run_data_collection.py", line 362, in <module>
    simulation.collect_simulation_data(Nreps=100, Nruns_per_energy=1000)
  File "/home/minseokhwan/xray_projects/xray_scattering_discrimination/run_data_collection.py", line 324, in collect_simulation_data
    simulation.collect_simulation_data(Nreps=100, Nruns_per_energy=1000)
  File "/home/minseokhwan/xray_projects/xray_scattering_discrimination/run_data_collection.py", line 324, in collect_simulation_data
    self.read_output(Nruns_per_energy)
  File "/home/minseokhwan/xray_projects/xray_scattering_discrimination/run_data_collection.py", line 254, in read_output
    self.read_output(Nruns_per_energy)
  File "/home/minseokhwan/xray_projects/xray_scattering_discrimination/run_data_collection.py", line 254, in read_output
    dE_trans[i] = E_inc - E_trans
ValueError: setting an array element with a sequence.
    dE_trans[i] = E_inc - E_trans
ValueError: setting an array element with a sequence.
TypeError: only length-1 arrays can be converted to Python scalars

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/minseokhwan/xray_projects/xray_scattering_discrimination/run_data_collection.py", line 362, in <module>
    simulation.collect_simulation_data(Nreps=100, Nruns_per_energy=1000)
  File "/home/minseokhwan/xray_projects/xray_scattering_discrimination/run_data_collection.py", line 324, in collect_simulation_data
    self.read_output(Nruns_per_energy)
  File "/home/minseokhwan/xray_projects/xray_scattering_discrimination/run_data_collection.py", line 254, in read_output
    dE_trans[i] = E_inc - E_trans
ValueError: setting an array element with a sequence.
TypeError: only length-1 arrays can be converted to Python scalars

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/minseokhwan/xray_projects/xray_scattering_discrimination/run_data_collection.py", line 362, in <module>
    simulation.collect_simulation_data(Nreps=100, Nruns_per_energy=1000)
  File "/home/minseokhwan/xray_projects/xray_scattering_discrimination/run_data_collection.py", line 324, in collect_simulation_data
    self.read_output(Nruns_per_energy)
  File "/home/minseokhwan/xray_projects/xray_scattering_discrimination/run_data_collection.py", line 254, in read_output
    dE_trans[i] = E_inc - E_trans
ValueError: setting an array element with a sequence.
